### ISSUE TITLE
Recurrent viral upper respiratory tract infections

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6172]
**Preferred term label:**
Recurrent viral upper respiratory tract infections

**Synonyms**
Recurrent viral URI

**Definition (free text, please give PubMed ID)**
An increased susceptibility to viral upper respiratory tract infections as manifested by a history of recurrent viral upper respiratory tract infections (running ears - otitis, sinusitis, pharyngitis, tonsillitis).
PMID: 30422556

**Parent term (use hpo.jax.org/app)**
Recurrent upper respiratory tract infections

**Comment by pnrobinson** [comment]
added as HP_0033166
